test/res_results_csv_comparisons.py

```diff
@@ -290,14 +326,41 @@ def regression_tables(self):
 
         print('Creating regression tables...')
         output_path = os.path.join(os.path.dirname(self.base_table_name), 'comparisons', 'deltas.csv')
-        feature_df.sub(base_df).round(2).transpose()[sorted_model_types].to_csv(output_path)
 
+        base_df = base_df.stack()
+        feature_df = feature_df.stack()
+
+        deltas = pd.DataFrame()
+        deltas['base'] = base_df
+        deltas['feature'] = feature_df
+        deltas['diff'] = deltas['feature'] - deltas['base']
+        deltas['% diff'] = 100*(deltas['diff']/deltas['base'])
+        deltas.fillna(0, inplace=True)
+        deltas = deltas.round(2)
+        
+        deltas.reset_index('build_existing_model.geometry_building_type_recs', inplace=True)
+        model_type_map = {'Single-Family Detached':'SFD', 'Single-Family Attached':'SFA', 'Multi-Family':'MF'}
+        deltas['build_existing_model.geometry_building_type_recs'] = deltas['build_existing_model.geometry_building_type_recs'].map(model_type_map)
+        deltas.to_csv(output_path)
 
 if __name__ == '__main__':
 
     # Inputs
-    base_table_name = sys.argv[1]
-    feature_table_name = sys.argv[2]
+    parser = argparse.ArgumentParser()
+    parser.add_argument("base_table_name", help="Filepath to base results table", type=str)
+    parser.add_argument("feature_table_name", help="Filepath to feature results table", type=str)
+    parser.add_argument("--use_cols", help="Which table's variable names to use (int)", type=int)  # Only can map from old to new right now
+    args = parser.parse_args()
```

Comment:
Can you include an example usage line?

Reply:
added in https://github.com/NREL/resstock/pull/554/commits/dea74b3970c2ddb05660255439da7e2b822e317b